devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -0,0 +1,173 @@
+		
+		// Samsung LED
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0406", outClusters: "0019", manufacturer: "Samsung Electronics", model: "SAMSUNG-ITM-Z-004", deviceJoinName: "ITM CPX Light", mnmn: "Samsung Electronics", vid: "SAMSUNG-ITM-Z-001"
+	}
+
+	// UI tile definitions
+	tiles(scale: 2) {
+		multiAttributeTile(name: "switch", type: "lighting", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState "on", label: '${name}', action: "switch.off", icon: "st.switches.light.on", backgroundColor: "#00A0DC", nextState: "turningOff"
+				attributeState "off", label: '${name}', action: "switch.on", icon: "st.switches.light.off", backgroundColor: "#ffffff", nextState: "turningOn"
+				attributeState "turningOn", label: '${name}', action: "switch.off", icon: "st.switches.light.on", backgroundColor: "#00A0DC", nextState: "turningOff"
+				attributeState "turningOff", label: '${name}', action: "switch.on", icon: "st.switches.light.off", backgroundColor: "#ffffff", nextState: "turningOn"
+			}
+			tileAttribute("device.level", key: "SLIDER_CONTROL") {
+				attributeState "level", action: "switch level.setLevel"
+			}
+		}
+		
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		
+		controlTile("colorTempSliderControl", "device.colorTemperature", "slider", width: 4, height: 2, inactiveLabel: false, range: "(2700..6500)") {
+			state "colorTemperature", action: "color temperature.setColorTemperature"
+		}
+		
+		main(["switch"])
+		details(["switch", "switchLevel", "colorTempSliderControl", "refresh"])
+	}
+}
+
+private getMOTION_CLUSTER() { 0x0406 }
+private getMOTION_STATUS_ATTRIBUTE() { 0x0000 }
+private getON_OFF_CLUSTER() { 0x0006 }
+private getCONFIGURE_REPORTING_RESPONSE() { 0x07 }
+private getON_DATA() { 0x01 }
+private getOFF_DATA() { 0x00 }
+
+def parse(String description) {
+	def event = zigbee.getEvent(description)
+	def zigbeeMap = zigbee.parseDescriptionAsMap(description)
+	
+	if (event) {
+		if (zigbeeMap.clusterInt == ON_OFF_CLUSTER && (zigbeeMap.data[0] != ON_DATA || zigbeeMap.data[0] != OFF_DATA)) {
+			return
+		}
+		
+		if (!(event.name == "level" && event.value == 0)) {
+			sendEvent(event)
+		}
+	} else {
+		def cluster = zigbee.parse(description)
+		
+		if (cluster && cluster.clusterId == ON_OFF_CLUSTER && cluster.command == CONFIGURE_REPORTING_RESPONSE) {
+			if (cluster.data[0] == 0x00) {
+				sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+			}
+		} else {
+			def zigbeeMap_child = zigbee.parseDescriptionAsMap(description)
```

Comment:
my suggestion was that you create an all new event 

`def event = createEvent(name: "motion", value: "active")`

`parseDescriptionAsMap` creates a populated map. You can see this because you're referencing `.clusterInt` on the next line.

Reply:
OK, I got it! I fixed that.